dependencyManagement/dependencyManagement.gradle.kts

```diff
@@ -88,17 +88,17 @@ val DEPENDENCIES = listOf(
   "commons-collections:commons-collections:3.2.2",
   "commons-digester:commons-digester:2.1",
   "commons-fileupload:commons-fileupload:1.4",
-  "commons-io:commons-io:2.9.0",
+  "commons-io:commons-io:2.10.0",
   "commons-lang:commons-lang:2.6",
   "commons-logging:commons-logging:1.2",
   "commons-validator:commons-validator:1.7",
   "info.solidsoft.spock:spock-global-unroll:0.5.1",
   "io.netty:netty:3.10.6.Final",
-  "org.assertj:assertj-core:3.19.0",
-  "org.awaitility:awaitility:4.0.3",
-  "org.checkerframework:checker-qual:3.13.0",
+  "org.assertj:assertj-core:3.20.0",
```

Comment:
Turns out assertj-core:3.20.0 is incompatible with previous release (they change return types of a bunch of methods)

Reply:
Hmm - I'll remember this if I ever feel like breaking ABI in sdk-testing! 🦊